metaflow/plugins/kfp/tests/flows/resilient_flow.py

```diff
@@ -32,6 +57,11 @@ def start(self):
             # this gives the test extra resilience.
             time.sleep(60 * 5)
         else:
+            validate_checkpoint_root(checkpoint_paths.root)
+            assert checkpoint_paths.resume_path.endswith(checkpoint_name)
+
+            obj = S3(s3root=checkpoint_paths.root).get(checkpoint_name).text
+            assert obj == checkpoint_obj
             print("let's succeed")
```

Comment:
This print doesn't seem helpful?

Reply:
it helps in the sense that there is step output: nice when running and debugging, rather than nothing 
added more to the print statement to double as comments

Reply:
thanks for catch!